chsdi/models/bod.py

```diff
@@ -320,63 +317,46 @@ class ServiceMetadataEn(Base, ServiceMetadata):
-                {'level': 0, 'resolution': 4000, 'scale': 14285750.5715, 'width': 1, 'height': 1},
-                {'level': 1, 'resolution': 3750, 'scale': 13392891.1608, 'width': 1, 'height': 1},
-                {'level': 2, 'resolution': 3500, 'scale': 12500031.7501, 'width': 1, 'height': 1},
-                {'level': 3, 'resolution': 3250, 'scale': 11607172.3393, 'width': 1, 'height': 1},
-                {'level': 4, 'resolution': 3000, 'scale': 10714312.9286, 'width': 1, 'height': 1},
-                {'level': 5, 'resolution': 2750, 'scale': 9821453.51791, 'width': 1, 'height': 1},
-                {'level': 6, 'resolution': 2500, 'scale': 8928594.10719, 'width': 1, 'height': 1},
-                {'level': 7, 'resolution': 2250, 'scale': 8035734.69647, 'width': 1, 'height': 1},
-                {'level': 8, 'resolution': 2000, 'scale': 7142875.28575, 'width': 1, 'height': 1},
-                {'level': 9, 'resolution': 1750, 'scale': 6250015.87503, 'width': 2, 'height': 1},
-                {'level': 10, 'resolution': 1500, 'scale': 5357156.46431, 'width': 2, 'height': 1},
-                {'level': 11, 'resolution': 1250, 'scale': 4464297.05359, 'width': 2, 'height': 1},
-                {'level': 12, 'resolution': 1000, 'scale': 3571437.64288, 'width': 2, 'height': 2},
-                {'level': 13, 'resolution': 750, 'scale': 2678578.23216, 'width': 3, 'height': 2},
-                {'level': 14, 'resolution': 650, 'scale': 2321434.46787, 'width': 3, 'height': 2},
-                {'level': 15, 'resolution': 500, 'scale': 1785718.82144, 'width': 4, 'height': 3},
-                {'level': 16, 'resolution': 250, 'scale': 892859.410719, 'width': 8, 'height': 5},
-                {'level': 17, 'resolution': 100, 'scale': 357143.764288, 'width': 19, 'height': 13},
-                {'level': 18, 'resolution': 50, 'scale': 178571.882144, 'width': 38, 'height': 25},
-                {'level': 19, 'resolution': 20, 'scale': 71428.7528575, 'width': 94, 'height': 63},
-                {'level': 20, 'resolution': 10, 'scale': 35714.3764288, 'width': 188, 'height': 125},
-                {'level': 21, 'resolution': 5, 'scale': 17857.1882144, 'width': 375, 'height': 250},
-                {'level': 22, 'resolution': 2.5, 'scale': 8928.59410719, 'width': 750, 'height': 500},
-                {'level': 23, 'resolution': 2, 'scale': 7142.87528575, 'width': 938, 'height': 625},
-                {'level': 24, 'resolution': 1.5, 'scale': 5357.15646431, 'width': 1250, 'height': 834},
-                {'level': 25, 'resolution': 1, 'scale': 3571.43764288, 'width': 1875, 'height': 1250},
-                {'level': 26, 'resolution': 0.5, 'scale': 1785.71882144, 'width': 3750, 'height': 2500},
-                {'level': 27, 'resolution': 0.25, 'scale': 892.857, 'width': 7500, 'height': 5000},
-                {'level': 28, 'resolution': 0.1, 'scale': 357.1425, 'width': 18750, 'height': 12500}
-            ]
+            'compressionQuality': None,
```

Comment:
The new lods are different than the old ones. Differences are in the 'scale' value only. When I create them as above with gaTileGrid, I get the following:

{'height': 1, 'width': 1, 'scale': 15118080.0, 'resolution': 4000.0, 'level': 0}
{'height': 1, 'width': 1, 'scale': 14173200.0, 'resolution': 3750.0, 'level': 1}
{'height': 1, 'width': 1, 'scale': 13228320.0, 'resolution': 3500.0, 'level': 2}
{'height': 1, 'width': 1, 'scale': 12283439.999999998, 'resolution': 3250.0, 'level': 3}
{'height': 1, 'width': 1, 'scale': 11338559.999999998, 'resolution': 3000.0, 'level': 4}
{'height': 1, 'width': 1, 'scale': 10393680.0, 'resolution': 2750.0, 'level': 5}
{'height': 1, 'width': 1, 'scale': 9448800.0, 'resolution': 2500.0, 'level': 6}
{'height': 1, 'width': 1, 'scale': 8503920.0, 'resolution': 2250.0, 'level': 7}
{'height': 1, 'width': 1, 'scale': 7559040.0, 'resolution': 2000.0, 'level': 8}
{'height': 1, 'width': 2, 'scale': 6614160.0, 'resolution': 1750.0, 'level': 9}
{'height': 1, 'width': 2, 'scale': 5669279.999999999, 'resolution': 1500.0, 'level': 10}
{'height': 1, 'width': 2, 'scale': 4724400.0, 'resolution': 1250.0, 'level': 11}
{'height': 2, 'width': 2, 'scale': 3779520.0, 'resolution': 1000.0, 'level': 12}
{'height': 2, 'width': 3, 'scale': 2834639.9999999995, 'resolution': 750.0, 'level': 13}
{'height': 2, 'width': 3, 'scale': 2456688.0, 'resolution': 650.0, 'level': 14}
{'height': 3, 'width': 4, 'scale': 1889760.0, 'resolution': 500.0, 'level': 15}
{'height': 5, 'width': 8, 'scale': 944880.0, 'resolution': 250.0, 'level': 16}
{'height': 13, 'width': 19, 'scale': 377951.99999999994, 'resolution': 100.0, 'level': 17}
{'height': 25, 'width': 38, 'scale': 188975.99999999997, 'resolution': 50.0, 'level': 18}
{'height': 63, 'width': 94, 'scale': 75590.4, 'resolution': 20.0, 'level': 19}
{'height': 125, 'width': 188, 'scale': 37795.2, 'resolution': 10.0, 'level': 20}
{'height': 250, 'width': 375, 'scale': 18897.6, 'resolution': 5.0, 'level': 21}
{'height': 500, 'width': 750, 'scale': 9448.8, 'resolution': 2.5, 'level': 22}
{'height': 625, 'width': 938, 'scale': 7559.039999999999, 'resolution': 2.0, 'level': 23}
{'height': 834, 'width': 1250, 'scale': 5669.279999999999, 'resolution': 1.5, 'level': 24}
{'height': 1250, 'width': 1875, 'scale': 3779.5199999999995, 'resolution': 1.0, 'level': 25}
{'height': 2500, 'width': 3750, 'scale': 1889.7599999999998, 'resolution': 0.5, 'level': 26}
{'height': 5000, 'width': 7500, 'scale': 944.8799999999999, 'resolution': 0.25, 'level': 27}
{'height': 12500, 'width': 18750, 'scale': 377.952, 'resolution': 0.1, 'level': 28}

getScale of gaTileGrid is also used in the WMTSGetCapabilities documents - so we really need to be sure with this.